### PR TITLE
resip/stack: improve p_received handling  without value

### PR DESCRIPTION
**Pull request by vak-oskh**
The Via parameter "received" is very similar to "rport". Neither RFC3581 nor RFC3261 exactly describes the "received" parameter with no value, while does not reject this possibility. Some User Agents treat both parameters similarly and send an empty "received" parameter in a request.

### CODE REVIEW

File: resip/stack/ReceivedParameter.cxx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      try
+      {
+         pb.skipChar();
+         pb.skipWhitespace();
+
+         // Handle cases such as ";received="
+         if(terminators[(unsigned char)(*pb.position())])
+         {
+            ErrLog(<< "Empty value in string-type parameter.");
+            DebugLog(<< "Define p_received with no value");
```
Comment (sgodin) [inline resip/stack/ReceivedParameter.cxx:29]: What is this DebugLog?  There is already and ErrLog, what purpose does the DebugLog serve?  What is the text of the debug log intending to communicate?  In particular what did you mean by "Define"?

**Comment by sgodin** [comment]
Just the one logging comment - otherwise looks sane to me.  Note:  I'm on vacation starting now and all next week.  So I likely won't be responsive during this.

File: resip/stack/ReceivedParameter.cxx (hunk 1 above)
Comment (vak-oskh) [reply resip/stack/ReceivedParameter.cxx]: Sorry, it was my investigation/testing log. I have removed it.

**Comment by vak-oskh** [comment]
> Just the one logging comment - otherwise looks sane to me. Note: I'm on vacation starting now and all next week. So I likely won't be responsive during this.

Thank you for the notice. Have a great vacation!

**Pull request merged**